platform/core.windows/src/org/netbeans/core/windows/view/dnd/DragWindow.java

```diff
@@ -95,15 +102,17 @@ private BufferedImage createContentImage( Component c, Dimension contentSize ) {
         GraphicsConfiguration config = GraphicsEnvironment.getLocalGraphicsEnvironment()
                     .getDefaultScreenDevice().getDefaultConfiguration();
 
-        BufferedImage res = config.createCompatibleImage(contentSize.width, contentSize.height);
+        BufferedImage res = config.createCompatibleImage(
+                contentSize.width * DPI_SCALE, contentSize.height * DPI_SCALE);
         Graphics2D g = res.createGraphics();
+        g.scale(DPI_SCALE, DPI_SCALE);
```

Comment:
this would be quite a large image on 4k screens. You sure its worth it to double the size to have the scaled down version look better?

Reply:
In the default configuration, it's a small thumbnail rectangle as seen in the screenshot in the PR, not a full-size copy of the component.

But looking at DragAndDropFeedbackVisualizer, which instantiates the DragWindow, there is a WinSysPrefs.DND_SMALLWINDOWS option that can be turned off to get the full-size window. If a platform app happens to be using that setting, it's true that the buffer could be several megabytes.

Should I make some efforts to skip the scaling in the WinSysPrefs.DND_SMALLWINDOWS=false case?

Reply:
oh nevermind. I assumed `contentSize` is the size of the component. Its only 250x250 on 1080p so this is no problem to double it.